code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -84,28 +84,31 @@
 			C = owner.client
 		else
 			stage = 2 // Let's try again later.
-			polling = 0
+			polling = FALSE
 			return
 
 		var/overlay = image('icons/mob/alien.dmi', loc = owner, icon_state = "burst_lie")
 		owner.overlays += overlay
 
 		spawn(6)
+			owner.overlays -= overlay
 			var/mob/living/carbon/alien/larva/new_xeno = new(owner.drop_location())
 			new_xeno.key = C.key
 			if(SSticker && SSticker.mode)
 				SSticker.mode.xenos += new_xeno.mind
 			new_xeno.mind.name = new_xeno.name
 			new_xeno.mind.assigned_role = SPECIAL_ROLE_XENOMORPH
 			new_xeno.mind.special_role = SPECIAL_ROLE_XENOMORPH
-			new_xeno << sound('sound/voice/hiss5.ogg',0,0,0,100)//To get the player's attention
+			SEND_SOUND(new_xeno, sound('sound/voice/hiss5.ogg'))
 			to_chat(new_xeno, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Xenomorph)</span>")
 
-			if(gib_on_success)
-				owner.gib()
-			else
+			if(burst_on_success) //If we burst naturally
+				owner.apply_damage(300, BRUTE, "chest")
+				owner.bleed(BLOOD_VOLUME_NORMAL)
+				var/obj/item/organ/external/chestorgans = owner.get_organ(ran_zone("chest"))
+				chestorgans.droplimb()
```

Comment:
this needs to explicitly break their chest. and, it shouldn't be random, and we have defines for these.
```suggestion
			if(burst_on_success) //If we burst naturally
				owner.apply_damage(300, BRUTE, "chest")
				owner.bleed(BLOOD_VOLUME_NORMAL)
				var/obj/item/organ/external/chestorgans = owner.get_organ(BODY_ZONE_CHEST)
				chestorgans.droplimb()
```

Reply:
Done and done.